lib/ruby_lsp/test_reporters/lsp_reporter.rb

```diff
@@ -14,11 +15,24 @@ class LspReporter
 
     #: -> void
     def initialize
+      dir_path = File.join(Dir.tmpdir, "ruby-lsp")
+      FileUtils.mkdir_p(dir_path) unless File.exist?(dir_path)
```

Comment:
No need to check first. If you're going to end up doing a syscall and IO op anyway, may as well just be the the `mkdir -p` call

```suggestion
      FileUtils.mkdir_p(dir_path)
```

Reply:
Good point